src/js/actions/transform.js

```diff
@@ -390,9 +431,9 @@ define(function (require, exports) {
                 positions: []
             },
             layerOneActions = _getMoveLayerActions
-                .call(this, document, layers.get(0), newPositions.get(0), payload.positions),
+                .call(this, document, layers.get(0), newPositions.get(0), "lt", payload.positions),
```

Comment:
Is "lt" a default value?  If so, can we just pass `undefined` here and default it down deeper?  This "hardcoded" value just seems out of place here.